main.css

```
@@ -3,6 +3,7 @@
   justify-content: center;
   align-items: center;
   height: 100vh;
+  padding: 0;
```

ohnoh ohohhhhh, q es ese padding??
```suggestion
  padding: 0;
```